src/components/ArticleDetails.jsx

```diff
@@ -1,15 +1,90 @@
 import { Link } from "react-router-dom";
+import { useState } from "react";
+import { voteArticleUp, voteArticleDown } from "../utils";
+
+export function ArticleDetails({ article, votes, article_id }) {
+  const [userVotes, setUserVotes] = useState(0);
+  const [errorState, setErrorState] = useState(false);
+
+  const handleClickUp = () => {
```

Comment:
This isn't a mandatory improvement but - your handleClickUp and handleClickDown functions look very similar and repeat the same code with different values - could you refactor them into one function? 🤔